lib/measured/measurable.rb

```diff
@@ -1,6 +1,8 @@
 class Measured::Measurable < Numeric
   include Measured::Arithmetic
 
+  PARSE_REGEX = /\A\s*([0-9.]+)\s*([a-zA-Z_-]+)\s*\Z/
```

Comment:
Finally, on that topic - we should consider commas and periods inter-changable. [A non-trivial amount of countries in the world represent the idea of `1.21` as `1,21`.](http://brilliantmaps.com/decimals/)